test_large_stdout_stream (performance) can occasionally fail
depending on the current build machine utilization.

--- pexpect-4.8.0/tests/test_performance.py
+++ pexpect-4.8.0/tests/test_performance.py
@@ -102,6 +102,7 @@ class PerformanceTestCase (PexpectTestCa
         self.faster_range(100000)
         print("100000 calls to faster_range:", (time.time() - start_time))
 
+    @unittest.skip("Skip performance test")
     def test_large_stdout_stream(self):
         e = pexpect.spawn('openssl rand -base64 {}'.format(1024*1024*25), searchwindowsize=1000)
         resp = e.expect(['Password:', pexpect.EOF, pexpect.TIMEOUT])
